cassandra-sink/README.adoc

```diff
@@ -1,7 +1,59 @@
-= Spring Cloud Stream Cassandra Sink
+//tag::ref-doc[]
+= Cassandra Sink (`cassandra`)
```

Comment:
I dropped the module name "(`cassandra`)" here, as this is a dataflow thing, and the user can register them as he wishes anyway